src/lib/eth-calls.ts

```diff
@@ -22,4 +22,23 @@ async function registerTable(
   return await tx.wait();
 }
 
-export { registerTable };
+async function runSql(
+  this: Connection,
+  tableId: number,
+  query: string
+): Promise<ContractReceipt> {
+  this.signer = this.signer ?? (await getSigner());
+  const address = await this.signer.getAddress();
+
+  const contractAddress = this.options.contract;
+
+  const contract = TablelandTables__factory.connect(
+    contractAddress,
+    this.signer
+  );
+  const tx = await contract.runSQL(address, tableId, query);
+
+  return await tx.wait();
```

Comment:
I have an Objective coming up after this to enable notifying the caller when tx receipts are materialized in the validator, but until then I just returned the receipt similar to the current `create` method.

Reply:
Actually the create method now "unpacks" this to some degree. I think its ok to return the raw receipt here though, because we're accessing the raw smart contract method anyway.